internal/profile/resource_profile.go

```diff
@@ -492,6 +492,14 @@ func (r ProfileResource) resetDefaultProfile(ctx context.Context, plan ProfileMo
 	defaultDevices := make(map[string]map[string]string)
 	defaultConfig := make(map[string]string)
 
+	// It is allowed to leave the project unspecified in the Terraform config,
+	// which defaults to the `default` project.
+	// For the GetProject call, the project name needs to be explicitly
+	// specified.
+	if projectName == "" {
```

Comment:
Would it not better to set the project value as `default` in the schema definition?

Reply:
You are right, this is way cleaner. At first, I was concerned, that setting the project explicitly to `default` would cause issues with the client (since this is the case for `target`), but this does not seam to be an issue. My manual tests have been successful.

I updated the PR.